public/video-ui/src/components/FormFields/ComposerTagPicker.js

```diff
@@ -5,9 +5,9 @@ import removeStringTagDuplicates from '../../util/removeStringTagDuplicates';
 import { keyCodes } from '../../constants/keyCodes';
 import UserActions from '../../constants/UserActions';
 
-export default class FormFieldBylinePicker extends React.Component {
+export default class ComposerTagPicker extends React.Component {
```

Comment:
I'm going to leave this because it picks tags for composer instead of youTube and there is a chance we are going to have to try to use tags from tagmanager. If this is not the case, I am happy to change it later.